runtime/shared_common/ClasspathManagerImpl2.cpp

```diff
@@ -1099,7 +1112,31 @@ SH_ClasspathManagerImpl2::validate(J9VMThread* currentThread, ROMClassWrapper* f
 	if (localFoundAtIndex == -1) {
 		/* We didn't find an exact classpath match. Walk the classpaths and check for validity. Note that the timestamp check is incorporated for efficiency. */
 		localFoundAtIndex = localValidate_CheckAndTimestampManually(currentThread, testCP, testCPIndex, compareTo, foundIdentified, &addToIdentified, staleItem);
+		/* If the classpath has just been identified by localValidate_CheckAndTimestampManually, store it in the identified array */
+		if (addToIdentified && (compareTo->getType()==CP_TYPE_CLASSPATH) && (*_runtimeFlagsPtr & J9SHR_RUNTIMEFLAG_ENABLE_LOCAL_CACHEING)) {
+			if (!releaseIdentifiedMutex) {
+				if (0 == _cache->enterLocalMutex(currentThread, _identifiedMutex, "identifiedMutex", "validate")) {
+					releaseIdentifiedMutex = true;
+				}
+			}
+			/* Identification succeeded. Store identified classpath. */
+			if (releaseIdentifiedMutex) {
+				if (local_StoreIdentified(currentThread, compareTo, testCP)==-1) {
```

Comment:
Pls add spaces and put constants on the left.

Reply:
Fixed.